_posts/2020-11-18-organising-raw-crystal.md

```diff
@@ -0,0 +1,71 @@
+* **Share knowledge and spark ideas**: we hope that the talks and conversations you'll have at the conference will make you want to start something new, or simply just tackle an old problem in a novel way.
+
+## The search for speakers
+After reading a few articles on the subject, I realised that finding speakers was going to be the most challenging part of the organisation. So before even thinking of a landing page for the conference, I launched [a simple Google form](https://forms.gle/tDSfrTQuWhSVmQo4A) to gather talk proposals.
+
+Only a handful of talks came through over the first week, so I decided to proactively get in touch with prospective speakers to prompt them to contribute.  
+I selected a few people based on their activity on the various Crystal channels - e.g. forum, gitter, github - and on their involvement in interesting projects - cherry-picked from shard aggregators websites such as [crystalshards.xyz](http://crystalshards.xyz).
+
+I also leveraged my network of Crystal friends. In particular, I asked Jack Thorne - of [Chicago Crystal](https://www.buzzsprout.com/1030945) fame - to look after and moderate a panel discussion to close the event - kudos Jack!
+<img src="{{ 'blog/2020-11-raw-crystal-jack.png' | asset_path }}" width="617"  class="center" alt="A couple of messages I exchanged Jack on Slack. I asked: would you be up for moderating a panel discussion at a conference, if we were to have one? He answered: yes!"/>
+
+In parallel, I engaged with a few members of the Crystal core team to secure a keynote speaker - thanks Brian - and look for collaborators in the organisation of the event.
+
+Finally, with the idea of ensuring a broad coverage of topics - from the language tool ecosystem to web application development - I decided to pick three featured speakers. You can find their names on the [event page](https://www.eventbrite.co.uk/e/raw-crystal-2020-tickets-127439094763), if you are curious.
+
+## Compromising on time
+Picking a time for the conference was quite challenging: on one hand, I wanted to make the event accessible to as many people across the world as possible. On the other hand, it is impossible to make everyone happy on a global scale.
+
+I decided to make a choice based on the following constraints:
+* the selected schedule should be friendly to people based in Argentina - that's where many members of the core team are based
+* the selected schedule should not affect my circadian rhythm too badly - I'm based in London, UK
+* the total span of the event should be manageable for a single person to run - so no marathons
+
+I also reviewed geo data related to users of my personal blog and to users of my YouTube channel. This reminded me that Crystal has a significant number of friends in North America.
+
+Based on all the above, I decided to go for a 6-hour-long conference starting at 8 am PST.
```

Comment:
I'd strongly suggest to communicate times (exclusively) in UTC. Everyone should easily now their respective UTC offset, but not the offset of some random other time zone (PST).
```suggestion
Based on all the above, I decided to go for a 6-hour-long conference starting at 16:00 UTC.
```

Reply:
(Optionally, a supplementary list of the observed time in other selected locations (like those mentioned elsewhere in the article) could be convenient)